todo_app/data/item.py

```diff
@@ -0,0 +1,73 @@
+from todo_app.config.credentials import *
+from todo_app.data.list import List
+import requests
+
+
+class Item:
+
+    def __init__(self, id, name, description, due, list_name, list_id):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.due = due
+        self.list_name = list_name
+        self.list_id = list_id
+
+    def create_Card(list_name, card_name):
+        """
+            Create a card as part of a list with the specified list name and card name.
+
+        Args:
+            list_name: List name.
+            card_name: Card name.
+        """
+        card = {
+            "name": card_name,
+            "idBoard": "61c9efb12a4989214a75b4a5",
```

Comment:
Use the `board_id` variable that was loaded from an env var.

Reply:
Fixed